Description: Remove cabal-th dependency
 This is only used to embed the version of the package, but since
 cabal-th is available just on a few architectures, it makes more sense
 to hard-code that and thus make shelltestrunner available on more
 architectures.
Author: Iustin Pop <iustin@debian.org>
Forwarded: no
Last-Update: 2012-03-26
--- a/shelltest.hs
+++ b/shelltest.hs
@@ -41,14 +41,14 @@
 import qualified System.FilePath.Find as Find (extension)
 import Control.Applicative ((<$>))
 import Data.Algorithm.Diff
-import Distribution.PackageDescription.TH (packageVariable, package, pkgVersion)
 
 strace :: Show a => a -> a
 strace a = trace (show a) a
 
 progname, version, progversion :: String
 progname = "shelltest"
-version = $(packageVariable (pkgVersion . package))
+-- sync with the cabal file
+version = "1.2.1"
 progversion = progname ++ " " ++ version
 proghelpsuffix :: [String]
 proghelpsuffix = [
--- a/shelltestrunner.cabal
+++ b/shelltestrunner.cabal
@@ -28,7 +28,6 @@
   ghc-options:    -threaded -W -fwarn-tabs
   build-depends:
                  base                 >= 4     && < 5
-                ,cabal-file-th
                 ,filemanip            >= 0.3   && < 0.4
                 ,HUnit                            < 1.3
                 ,cmdargs              >= 0.7   && < 0.10
